autopush/router/webpush.py

```diff
@@ -72,7 +73,8 @@ def preflight_check(self, uaid, channel_id):
             self.ap_settings.message_tables[month_table].all_channels,
             uaid=uaid)
 
-        if not exists or channel_id not in chans:
+        if (not exists or channel_id.lower() not
+                in map(lambda x: normalize_id(x), chans)):
```

Comment:
Yep. We're kinda stuck at the moment because of legacy records. Eventually, they should expire out, but for now, we have to account for them.